<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchFrom</class>
 <widget class="QWidget" name="SearchFrom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_bg" native="true">
     <widget class="QWidget" name="widget_h" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #09A3DC;</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>0</y>
         <width>51</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Segoe UI Black&quot;;</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_h">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
       </property>
       <property name="text">
        <string>添加好友</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_m" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>401</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QPushButton" name="pushButton_sendadd">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #09A3DC;
color: rgb(255, 255, 255);
font: 9pt &quot;Adobe 黑体 Std R&quot;;</string>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>284</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_icon">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>75</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>75</height>
           </size>
          </property>
          <property name="text">
           <string>用户不存在</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Adobe 黑体 Std R&quot;;</string>
            </property>
            <property name="text">
             <string>Name：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Adobe 黑体 Std R&quot;;</string>
            </property>
            <property name="text">
             <string>Desc：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Adobe 黑体 Std R&quot;;</string>
            </property>
            <property name="text">
             <string>Id:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_name">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Adobe 黑体 Std R&quot;;</string>
            </property>
            <property name="text">
             <string>—————</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_desc">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Adobe 黑体 Std R&quot;;</string>
            </property>
            <property name="text">
             <string>—————</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_id">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Adobe 黑体 Std R&quot;;</string>
            </property>
            <property name="text">
             <string>—————</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchFrom</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>725</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
